src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -0,0 +1,85 @@
+import { arrowKeysNavigation, getRowAndColumnCount } from '../handleKeyPress';
+
+const { JSDOM } = require('jsdom');
+
+describe('handle-key-press', () => {
+  describe('moveToNextFocus', () => {});
+
+  describe('arrowKeysNavigation', () => {
+    const evt = {};
+    const rowAndColumnCount = {};
+    let rowIndex;
+    let colIndex;
```

Comment:
I think you should add a `beforeEach`. That way you make sure everything is reset before each test plus you only need to change one or a few variables per test.

Reply:
good point. Fixed.